ocaml/parsing/jane_syntax.mli

```diff
@@ -238,6 +251,18 @@ module Expression : sig
     loc:Location.t -> attrs:Parsetree.attributes -> t -> Parsetree.expression
 end
 
+(** Novel syntax in cases *)
+module Case : sig
+  type t =
+    | Jcase_pattern_guarded of Pattern_guarded.case
+
+  include AST
+    with type t := t * Parsetree.attributes
```

Comment:
This attribute list is always empty, no? I think we should probably just return `t`, and not `t` together with attributes.